protoc-gen-go/generator/generator.go

```diff
@@ -92,6 +92,12 @@ func RegisterPlugin(p Plugin) {
 	plugins = append(plugins, p)
 }
 
+// An ImportPath is the import path of a Go package. e.g., "github.com/golang/protobuf".
```

Comment:
Nit: "google.golang.org/genproto/protobuf"

since "github.com/golang/protobuf" is not a go package.

Reply:
done